app/src/main/java/org/thoughtcrime/securesms/conversation/start/ContactListAdapter.kt

```diff
@@ -72,13 +85,9 @@ class ContactListAdapter(
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
         return if (viewType == ViewType.Contact) {
-            ContactViewHolder(
-                ViewContactBinding.inflate(LayoutInflater.from(context), parent, false)
-            )
+            ContactViewHolder(ViewContactBinding.inflate(LayoutInflater.from(context), parent, false))
         } else {
-            HeaderViewHolder(
-                ContactSectionHeaderBinding.inflate(LayoutInflater.from(context), parent, false)
-            )
+            HeaderViewHolder(ContactSectionHeaderBinding.inflate(LayoutInflater.from(context), parent, false))
```

Comment:
I was looking for functional changes in this file, but it appears to just be reformatting.

I guess we're all guilty of this, but it'd be nice to have in a separate refactoring PR if not necessary for this ticket, just because a refactoring PR is faster to review, as we're expecting the behaviour to remain the same.

Reply:
I see your point, but the problem is that we never have free time to perform (low priority) refactoring PRs - so if we left them for a separate PR they'd simply never get done. As such I believe that the lesser of two evils is to adjust stuff that's poorly formatted as we see it (i.e., try to leave each file we touch in just a little bit nicer shape than when we found it) and just accept that there might be some  noise in the PR.